src/components/inject.js

```diff
@@ -59,7 +59,7 @@ Please be sure the component that calls createSource or createToken is within an
     }
 
     context: Context;
-    wrappedInstance: ?WrappedComponent;
+    wrappedInstance: ?React.Component<Object, any>;
```

Comment:
Should this be `{}` instead of `Object`?

Reply:
Thanks! I think it should be `wrappedInstance: ?React.Component<{stripe: StripeProps} & Props, any>;`, right?

Reply:
Oh, yeah, true. I wasn't even thinking that hard :) Just don't like to see `Object`.